plugins/language_model/knn.cpp

```diff
@@ -690,6 +690,14 @@ namespace {
 
     auto lexicon = grn_tokenizer_query_get_lexicon(ctx, query);
     auto source_column = grn_tokenizer_query_get_source_column(ctx, query);
+    if (!source_column) {
+      GRN_PLUGIN_ERROR(ctx,
+                       GRN_TOKENIZER_ERROR,
+                       "%s source column not found. Note: This tokenizer does "
+                       "not support 'tokenize' command.",
```

Comment:
Does the message "This tokenizer does not support 'tokenize' command." appear because this error always occurs when the tokenize command is executed?